pkg/workload/workload.go

```diff
@@ -362,9 +362,9 @@ func GetQueueOrderTimestamp(w *kueue.Workload) *metav1.Time {
 	return &w.CreationTimestamp
 }
 
-// IsAdmitted checks if workload is admitted based on conditions
-func IsAdmitted(w *kueue.Workload) bool {
-	return apimeta.IsStatusConditionTrue(w.Status.Conditions, kueue.WorkloadAdmitted)
+// HasQuotaReservation checks if workload is admitted based on conditions
+func HasQuotaReservation(w *kueue.Workload) bool {
+	return apimeta.IsStatusConditionTrue(w.Status.Conditions, kueue.WorkloadQuotaReserved)
```

Comment:
just to be clear that my comment above is not misinterpreted:
This rename is fine.

Reply:
ok